content/blog/hackaton-2022-building-a-pong.md

```diff
@@ -0,0 +1,212 @@
+
+---
+Title: Hackapong fun at the shop!
+Date: 2022-11-04 13::55:00
+Author: Jean-Philippe Lamebrt - alex Hoguin
+Category: WDA
+Tags: hackathon, wda
+Slug: hackaton-2022-building-a-pong
+Status: published
+---
+## Here Comes the Hackapong 
+Our Yearly hackathon is a very important moment, first of all we use it to bond as a team and work as a team to use our tools and see what we could add in the future. We used this week to build on tools we already use or that are coming to WDA soon.
+
+Our team decided to put the WDA plugin functionnality to good use and we decided to build a pong game that would be playable between two callers using WDA. 
+
+To accomplish that feat we used existing tools from the platform, webrtc, asterisk, nginx, etc and some that are not yet available i.e. the plugin interface that is in the works WDA. 
+
+We started by finding a pong javascript [codebase](https://gist.github.com/straker/81b59eecf70da93af396f963596dfdc5) that was functional and had all the feature we needed. We used an open-source project and we built on that. 
+
+The issues we faced were mostly related to synchronisation and control of the paddles for the players. 
+
+## Intercommunication 
+
+Since we used DTMF's to control the paddles ( we wanted it to work with a plastic phone too ) we had to use the wazo-sdk javascript library. We linked the user control ( arrow up and arrow down ) to send DTMF's to the stack. 
```

Comment:
```suggestion
We wanted it to work with a plastic phone too, so we chose to use DTMFs to control the paddles. 
For this we had to use the wazo-sdk javascript library. We linked the user controls ( arrow up and arrow down ) to send DTMFs to the stack. 
```

Reply:
(Removed apostrophes for pluralized DTMF, those are to indicate possession not plural.)